feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -1,14 +1,31 @@
+import { runImporter } from "../importer";
+
 export default class DataAccount {
     constructor() {
-        this.importingResults = [];
+        this.importingReports = [];
```

Comment:
The problem with this basically empty DataAccount is that, first, it does not establish an API for imported files and so on. Second, since it is built in an incorrect (empty) state, every client function receiving it should have first to check for validity before working on it. In the previous installment we were hauling around the object, which was being modified as a side effect of using functions that returned results. I seem to understand that's no longer the case, since we're evaluating things lazily and we need to keep pointers in the object. So can we please consider this in this cycle?

As I said in other comment, again the issue is that this is a big change that will affect all the PRs, and is replicated in all PRs, so it's understandable to put it in a different PR, or at any rate in the first in the chain, #1190